src/sentry/options/defaults.py

```diff
@@ -214,3 +214,4 @@
 
 # Max number of tags to combine in a single query in Discover2 tags facet.
 register("discover2.max_tags_to_combine", default=3, flags=FLAG_PRIORITIZE_DISK)
+register("ingest.use-relay-dsn-sample-rate", default=0.0)
```

Comment:
Could you add a comment here about what it affects, and what 0/1 mean?

Reply:
_nit_: All of the ingest-related options are called `store.*`, could you update that?